### PR TITLE
Update ggcaller to 1.3.0

### PR DESCRIPTION
**Pull request by BiocondaBot**
Update [`ggcaller`](https://bioconda.github.io/recipes/ggcaller/README.html): **1.2.4** &rarr; **1.3.0**

[install with bioconda](http://bioconda.github.io/recipes/ggcaller/README.html) [Conda](https://anaconda.org/bioconda/ggcaller/files)

Info | Link or Description
-----|--------------------
Recipe | [`recipes/ggcaller`](https://github.com//bioconda/bioconda-recipes/tree/bump/ggcaller/recipes/ggcaller) (click to view/edit other files)
Summary | A de Bruijn graph-based gene-caller and pangenome analysis tool
Home | [https://github.com/samhorsfield96/ggCaller](https://github.com/samhorsfield96/ggCaller)
Releases |[https://github.com/samhorsfield96/ggCaller/tags](https://github.com/samhorsfield96/ggCaller/tags)
Author | @samhorsfield96
***

This pull request was automatically generated (see [docs](https://bioconda.github.io/contributor/updating.html)).

### CODE REVIEW

File: recipes/ggcaller/meta.yaml [hunk 1]
```diff
@@ -1,46 +1,62 @@
...
+    - pytorch
+    - boost-cpp
+    - setuptools
+    - llvm-openmp # [osx]
+    - libgomp # [linux]
+    - mkl  # [linux]
+    - lapack  # [osx]
+    - openblas  # [osx]
+    - tbb-devel
+    - tbb
```
Comment (johnlees) [inline recipes/ggcaller/meta.yaml:44]: I think these may not be needed here if already in `build`

File: recipes/ggcaller/meta.yaml [hunk 2]
```diff
@@ -1,46 +1,62 @@
...
-    - setuptools
-    - llvm-openmp  # [osx]
-    - libgomp      # [linux64]
+    - tbb
+    - tbb-devel
   host:
     - python
     - pybind11
-    - seqan3 <3.1
+    - eigen 3.3
```
Comment (johnlees) [inline recipes/ggcaller/meta.yaml:30]: Check version specification correct, should it be `>=`?

File: recipes/ggcaller/meta.yaml [hunk 3]
```diff
@@ -50,17 +66,29 @@ requirements:
     - xorg-libxpm
     - xorg-libxaw
     - xorg-libxfixes
+    - intbitset
+    - networkx
+    - uncertainties
+    - eigen 3.3
```
Comment (johnlees) [inline recipes/ggcaller/meta.yaml:72]: Same version comment as above

File: recipes/ggcaller/meta.yaml [hunk 4]
```diff
@@ -1,46 +1,62 @@
...
-    - pthread-stubs
     - zlib
+    - pip
     - bzip2
+    - pytorch
+    - boost-cpp
+    - setuptools
+    - llvm-openmp # [osx]
+    - libgomp # [linux]
+    - mkl  # [linux]
```
Comment (johnlees) [inline recipes/ggcaller/meta.yaml:40]: Double-check this is ok. What about non0

File: recipes/ggcaller/meta.yaml [hunk 5]
```diff
@@ -1,46 +1,62 @@
...
     - python
     - pybind11
-    - seqan3 <3.1
+    - eigen 3.3
     - bifrost
-    - pthread-stubs
     - zlib
+    - pip
     - bzip2
+    - pytorch
```
Comment (johnlees) [inline recipes/ggcaller/meta.yaml:35]: Just double-checking this is needed for build?

File: recipes/ggcaller/meta.yaml [hunk 6]
```diff
@@ -50,17 +66,29 @@ requirements:
...
+    - gffutils
+    - cd-hit
+    - python-wget
 
 test:
   imports:
     - ggCaller_cpp
-    - balrog
     - ggCaller
+    - panaroo_runner
```
Comment (johnlees) [inline recipes/ggcaller/meta.yaml:86]: Double-checking this name doesn't clash with anything from panaroo?

Comment (samhorsfield96) [reply recipes/ggcaller/meta.yaml:86]: I've checked the panaroo github and recipe on bioconda, no clashes that I could find.

File: recipes/ggcaller/meta.yaml (hunk 5 above)
Comment (samhorsfield96) [reply recipes/ggcaller/meta.yaml:35]: This is required for libtorch linking in CMakeLists.txt.

File: recipes/ggcaller/meta.yaml (hunk 3 above)
Comment (johnlees) [reply recipes/ggcaller/meta.yaml:72]: Actually I think you can just remove, eigen is a build time dependency?
```suggestion
```

**Review by johnlees — APPROVED** [review]

**Pull request merged**